src/main/java/com/toptrumps/core/DeckParser.java

```diff
@@ -1,4 +1,67 @@
 package com.toptrumps.core;
 
+
+import java.io.*;
+import java.util.ArrayList;
+
+
+// TODO: Remove this class
 public class DeckParser {
+
+    private ArrayList<Card> cards;
+
+    public DeckParser() {
+        buildDeck(); // calls the build deck method
```

Comment:
This comment is not really helpful. We can either start writing our [javadoc](https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html) right away or leave it for later!